#!/usr/bin/env python
PACKAGE = "test_pkg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("p1min_param",    int_t,    0, "p1min parameter", 0,  0, 260)
gen.add("p1max_param",    int_t,    0, "p1max parameter", 0,  0, 260)
gen.add("p2min_param",    int_t,    0, "p2min parameter", 0,  0, 260)
gen.add("p2max_param",    int_t,    0, "p2max parameter", 0,  0, 260)
gen.add("p3min_param",    int_t,    0, "p3min parameter", 100,  0, 260)
gen.add("p3max_param",    int_t,    0, "p3max parameter", 255,  0, 260)

exit(gen.generate(PACKAGE, "test_pkg", "basket"))


